Lib/test/test_httpservers.py

```diff
@@ -33,6 +36,8 @@
 from test.support import (
     is_apple, import_helper, os_helper, requires_subprocess, threading_helper
 )
+from test.support.script_helper import spawn_python, kill_python
```

Comment:
```suggestion
from test.support.script_helper import kill_python, spawn_python
```

I'm trying to keep imports sorted